pkg/cli/admin/upgrade/status/health.go

```diff
@@ -78,6 +80,7 @@ type impactType string
 // considered whether these are exactly the ones that we need.
 const (
 	noneImpactType                    impactType = "None"
+	alertImpactType                   impactType = "Alert"
```

Comment:
"Alert" is not an impact on the cluster though. This is a manifestation of a similar problem like https://issues.redhat.com/browse/OTA-1173 : we see an alert but do not know what it actually means.

Maybe we should have an impact type `Unknown` for now, now sure :(